src/keyboard.c

```diff
@@ -14,6 +14,8 @@ See the file LICENSE for details.
 
 #define KEY_INVALID 0127
 
+#define W_SUBSTITUTE -1
```

Comment:
This looks super sketchy... I'm trying to figure out if we can substitute the value of `KEY_INVALID` with `-1` or maybe `255`, do you have any insight?